back/engines/commercial/multi_tenancy/db/seeds/custom_forms.rb

```diff
@@ -6,33 +6,38 @@ module MultiTenancy
   module Seeds
     class CustomForms < Base
       def run
-        3.times do |_i|
-          process_type = %w[continuous timeline timeline].sample
-          project = case process_type
-          when 'timeline'
-            Phase.where(participation_method: %w[ideation budgeting]).all.sample&.project
-          when 'continuous'
-            Project.where(participation_method: %w[ideation budgeting]).all.sample
-          end
+        # This will add at least one custom form to existing projects
+        %w[continuous timeline].each do |process_type|
+          create_form(process_type, 'ideation')
+          create_form(process_type, 'budgeting')
+        end
+      end
 
-          next unless project
+      private
 
-          # TODO
+      def create_form(process_type, participation_method)
+        project = case process_type
+        when 'timeline'
+          Phase.where(participation_method: participation_method).all.sample&.project
+        when 'continuous'
+          Project.where(participation_method: participation_method).all.sample
+        end
 
-          # custom_form = project.custom_form || CustomForm.create!(participation_context: project)
-          # built_in_custom_field = IdeaCustomFieldsService.new(custom_form).find_or_build_field %w[title_multiloc body_multiloc location_description].sample
-          # built_in_custom_field.description_multiloc = runner.create_for_some_locales { Faker::Lorem.sentence }
-          # built_in_custom_field.save!
-          # next unless rand(3) == 0
+        return unless project && !project.custom_form
```

Comment:
I find `return if !project || project.custom_form` much more readable, but I guess it's a matter of taste 😅